www-apps/cgp/cgp-1.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit webapp
+
+MY_PN="${PN^^}"
+
+DESCRIPTION="A graphical web-based front-end for visualizing RRD collected by collectd"
+HOMEPAGE="https://pommi.nethuis.nl/category/cgp/"
+SRC_URI="https://github.com/pommi/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+KEYWORDS="~amd64 ~x86"
+IUSE="+json"
```

Comment:
what is the purpose to enforce the single flag by default?

Reply:
Hm. Yeah, i will remove it and depend on dev-lang/php[json]. Should be anyway the one, what a user wants.